btcec/schnorr/musig2/musig2_test.go

```diff
@@ -26,14 +27,49 @@ var (
 	key3Bytes, _ = hex.DecodeString("3590A94E768F8E1815C2F24B4D80A8E3149" +
 		"316C3518CE7B7AD338368D038CA66")
 
-	testKeys = [][]byte{key1Bytes, key2Bytes, key3Bytes}
-
-	keyCombo1, _ = hex.DecodeString("E5830140512195D74C8307E39637CBE5FB730EBEAB80EC514CF88A877CEEEE0B")
-	keyCombo2, _ = hex.DecodeString("D70CD69A2647F7390973DF48CBFA2CCC407B8B2D60B08C5F1641185C7998A290")
-	keyCombo3, _ = hex.DecodeString("81A8B093912C9E481408D09776CEFB48AEB8B65481B6BAAFB3C5810106717BEB")
-	keyCombo4, _ = hex.DecodeString("2EB18851887E7BDC5E830E89B19DDBC28078F1FA88AAD0AD01CA06FE4F80210B")
+	invalidPk1, _ = hex.DecodeString("00000000000000000000000000000000" +
+		"00000000000000000000000000000005")
+	invalidPk2, _ = hex.DecodeString("FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF" +
+		"FFFFFFFFFFFFFFFFFFFFFFFEFFFFFC30")
+	invalidTweak, _ = hex.DecodeString("FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFE" +
+		"BAAEDCE6AF48A03BBFD25E8CD0364141")
+
+	testKeys = [][]byte{key1Bytes, key2Bytes, key3Bytes, invalidPk1,
+		invalidPk2}
+
+	keyCombo1, _ = hex.DecodeString("E5830140512195D74C8307E39637CBE5FB73" +
+		"0EBEAB80EC514CF88A877CEEEE0B")
+	keyCombo2, _ = hex.DecodeString("D70CD69A2647F7390973DF48CBFA2CCC407B" +
+		"8B2D60B08C5F1641185C7998A290")
+	keyCombo3, _ = hex.DecodeString("81A8B093912C9E481408D09776CEFB48AEB8" +
+		"B65481B6BAAFB3C5810106717BEB")
+	keyCombo4, _ = hex.DecodeString("2EB18851887E7BDC5E830E89B19DDBC28078" +
+		"F1FA88AAD0AD01CA06FE4F80210B")
 )
 
+// getInfinityTweak returns a tweak that, when tweaking G_, triggers
+// the ErrTweakedKeyIsInfinity error.
+func getInfinityTweak() KeyTweakDesc {
+	generator := btcec.Generator()
+
+	keySet := []*btcec.PublicKey{generator}
+
+	keysHash := keyHashFingerprint(keySet, true)
+	uniqueKeyIndex := secondUniqueKeyIndex(keySet, true)
+
+	n := &btcec.ModNScalar{}
+
+	n.SetByteSlice(invalidTweak)
+
+	coeff := aggregationCoefficient(keySet, generator, keysHash,
```

Comment:
nit: formatting of multi-line function call.

Reply:
like so?
```golang
coeff := aggregationCoefficient(
		keySet, generator, keysHash, uniqueKeyIndex,
	).Negate().Add(n)
```

Reply:
Yes, that looks good to me :+1: